extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.icons.js

```diff
@@ -0,0 +1,63 @@
+define(
+	'wikia.articleVideo.featuredVideo.jwplayer.icons',
+	['wikia.articleVideo.featuredVideo.jwplayer.instance'],
+	function (playerInstance) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "playerInstance". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)